Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MaxTextLength.swift
+//  Mobile Test Runner for Jama
+//
+//  Created by PSU1 on 7/5/17.
+//  Copyright © 2017 Jaca. All rights reserved.
+//
+
+import Foundation
```

Comment:
I moved the file to utils, but don't know what\how to write tests for it. the code adds a max length field in the attributes inspector in Xcode. so it doesn't effect our app but Xcode.